sky/serve/load_balancer.py

```diff
@@ -91,7 +91,21 @@ async def _redirect_handler(self, request: fastapi.Request):
         logger.info(f'Redirecting request to {path}')
         return fastapi.responses.RedirectResponse(url=path)
 
+    async def _get_urls(self, request: fastapi.Request):
+        del request  # Unused
+
+        ready_replica_urls = self._load_balancing_policy.ready_replicas
```

Comment:
Maybe we should cache this in this SkyServeLoadBalancer class whenever we've finished a sync. Seems appropriate.

Alternatively, we should make abstract class LoadBalancingPolicy expose a get_ready_replicas() and use it here? Subclasses may want to have R/W locks to protect this field.